lib/fs.js

```diff
@@ -40,6 +41,8 @@ const isWindows = process.platform === 'win32';
 
 const errnoException = util._errnoException;
 
+const noop = function() {};
+
 var printDeprecation;
 try {
   printDeprecation = require('internal/util').printDeprecationMessage;
```

Comment:
You can put `var` right here (to avoid duplication) or change top-level declaration to `let`.